R/epidist.R

```diff
@@ -1,11 +1,30 @@
-#' Parameter probability mass function
+#' Distribution (PDF, CDF, PMF) and its parameters for epidemiological 
```

Comment:
There are "trailing whitespace" warnings from `lintr` on a few lines, perhaps a `styler` run is called for?

Reply:
Resolved all the warnings from `lintr` (at least locally).